src/Common/Helpers/Constants.cs

```diff
@@ -37,7 +37,12 @@ public static class Constants
         public const string EventHubEntities = "Event Hubs";
         public const string NotificationHubEntities = "Notification Hubs";
 
-
+        public const string ActiveMessages = "Active";
+        public const string DeadLetterMessages = "DeadLetter";
+        public const string ScheduledMessages = "Scheduled";
+        public const string TransferMessages = "Transfer";
```

Comment:
These are messages being transfered. I wonder if the name should reflect that.

```suggestion
        public const string TransferMessages = "InTransfer";
```

Reply:
Seems like a good idea to me.